src/axom/slic/core/LogStream.hpp

```diff
@@ -102,7 +104,20 @@ class LogStream
                       bool filter_duplicates) = 0;
 
   /*!
-   * \brief Flushes the log stream. It's a NO-OP by default.
+   * \brief Flushes the log stream on current rank. It's a NO-OP by default.
```

Comment:
Is this clear enough that it is flushing localing to the output stream and not down the SLIC stream? I tried to word it a couple times but it wasn't helping.  I could spend more time on it if you like.

Reply:
> Is this clear enough that it is flushing localing to the output stream and not down the SLIC stream? I tried to word it a couple times but it wasn't helping. I could spend more time on it if you like.

I gave it another pass, and used the term "console" to try and make it more clear that it is going directly to the output stream.